pkg/server/server.go

```diff
@@ -70,12 +76,25 @@ func (p PizzaOvenServer) handleRequest(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = p.processRepository(data.URL)
-	if err != nil {
-		p.Logger.Errorf("Could not process repository input: %v with error: %v", r.Body, err)
-		http.Error(w, "Could not process input", http.StatusInternalServerError)
-		return
+	w.WriteHeader(http.StatusAccepted)
+	if p.Config.Wait {
```

Comment:
Thanks for this awesome PR!! Great work and looks well thought out.

I think this misses the mark just abit: individual clients that connect to the server should be able to set the time to live for their own connection. This config setting will _always_ be enforced (since it is set via a flag at startup of the service) regardless of if the user wants to keep the connection alive or not.

Instead, we should inspect some field on the input data:
```json
{
    "url": "https://some-git-repo.com",
    "ttl": 300
}
```
to wait 300 seconds before the connection is cut.

The default behavior if that field is not provided should be to close the connection: i.e., don't wait for the processing to finish.

Reply:
thanks for the feedback! I think I understand know, this needs to work for the pizza-cli with something like:

```bash
pizza bake url ttl
```
So, that each client enforces the ttl, rather than the service. But as discussed in the issue.

>Maybe a better behavior is for a kubectl like experience with the --wait flag / command:
>
>by default, the server validates the request and returns the status. It processes commits in the background.
>if a --wait flag / request json field is set, the server keeps the connection alive until processing is completed or fails.

The wait convention would be better, right? 
```
{
    "url": "https://some-git-repo.com",
    "wait": true
}
```
I pushed some changes, let me know if this is good.